design/media_types.go

```diff
@@ -199,40 +199,40 @@ var User = a.MediaType("application/vnd.user+json", func() {
 	})
 })
 
-var searchPagingLinks = Type("searchPagingLinks", func() {
-	Attribute("prev", String)
-	Attribute("next", String)
-	Attribute("first", String)
-	Attribute("last", String)
+var searchPagingLinks = a.Type("searchPagingLinks", func() {
+	a.Attribute("prev", d.String)
```

Comment:
As a follow-up on the PR @kwk did: I think we should standardize on how to do the import renames: I would propose 3-letter lowercase, like "dsl" or "api"